clang-tools-extra/docs/clang-tidy/checks/modernize/use-std-numbers.rst

```diff
@@ -0,0 +1,28 @@
+.. title:: clang-tidy - modernize-use-std-numbers
+
+modernize-use-std-numbers
+=========================
+
+Finds constants and function calls to math functions that can be replaced
+with c++20's mathematical constants from the ``numbers`` header and offers fix-it hints.
+Does not match the use of variables with that value, and instead,
+offers a replacement at the definition of those variables.
+
```

Comment:
List here all possible replacements.

Reply:
done

Reply:
I’ve also extended the documentation slightly to explain the replacements a bit further.